procedures/foreman/branch.md.erb

```diff
@@ -29,10 +29,16 @@
 
 - [ ] Ensure headline features planned for the release have been merged or push them off to the next release.
 - Translations:
+  - [ ] Setup Transifex CLI if not already installed: Follow the [Transifex CLI documentation](https://developers.transifex.com/docs/cli) for installation and authentication setup
   - [ ] Update locales in foreman __develop__: `make -C locale tx-update`
-  - [ ] Ask plugin authors to start extracting i18n strings and pushing the changes into develop/master git branches so Transifex can pick it up
-  - [ ] Announce string freeze date on Discourse and [send announcement](https://www.transifex.com/foreman/communication/?q=project%3Aforeman)
-  - [ ] Update [foreman-dev](https://community.theforeman.org/c/development) with [translations status](https://www.transifex.com/foreman/foreman/languages/) (click on Report) to encourage 100% translations before release
+  - [ ] Coordinate with plugin authors to extract i18n strings:
+    - [ ] Ask plugin authors to start extracting i18n strings and pushing the changes into develop/master git branches so Transifex can pick it up (send a message in the Matrix channel)
+  - [ ] Work with Transifex for translation status and announcements:
+    - [ ] Export translation statistics from [Transifex](https://app.transifex.com/foreman/foreman/languages/) (click on Report) and save as `foreman_foreman.languages.csv`. See [this](https://help.transifex.com/en/articles/6304029-progress-report#h_03f3e11758) help article on how to generate the report for all languages.
+    - [ ] Post string freeze announcement to [foreman-dev](https://community.theforeman.org/c/development) using `<%= rel_eng_script('string_freeze_announcement') %> foreman_foreman.languages.csv <branching_date> <rc1_date> <rc2_date> <planning_url>` to encourage 100% translations before release
```

Comment:
Does putting it in brackets work? I'm worried that they're interpreted as HTML tags and avoided that elsewhere.